lib/routes/163/utils.ts

```diff
@@ -3,18 +3,16 @@ const __dirname = getCurrentPath(import.meta.url);
 
 import got from '@/utils/got';
 import { load } from 'cheerio';
-import iconv from 'iconv-lite';
 import { art } from '@/utils/render';
 import * as path from 'node:path';
 
-const parseDyArticle = (charset, item, tryGet) =>
+const parseDyArticle = (item, tryGet) =>
```

Comment:
https://github.com/DIYgod/RSSHub/blob/6a2f748d24074f22d158f7234990886e337c4115/lib/routes/163/dy.ts#L43
should also be updated